go/interpreter/lfvm/instructions.go

```diff
@@ -1084,19 +1091,20 @@ func opCall(c *context) {
 	// and not doing it would be identified by the replay tool as an error.
 	toAddr := tosca.Address(addr.Bytes20())
```

Comment:
Can you move these two lines somewhere else? The current code section is computing gas costs, and then this conversion is declared in between.  

IMO this belongs between stack pops & gas cost or after gas costs